### ISSUE TITLE
ampserver.py divide is broken

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3288]
|[<img alt="jdavisp3's avatar" src="https://avatars.githubusercontent.com/u/43582?s=50" width="50" height="50">](https://github.com/jdavisp3)| @jdavisp3 reported|
|-|-|
|Trac ID|trac#3288|
|Type|enhancement|
|Created|2008-06-08 00:52:38Z|

The ampserver.py example code is broken for the divide case.

1. The variable is misnamed at one point.

2. The result isn't necessarily a floating point value.

Attachments:

* [0001-Fix-example.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a06/a0652c304b553d23fef50b4845390fb7a11d2107/de1de5c2a2937997f8f1a41ab8607a11ac5ee4b7.patch) (888 bytes) - added by davep on 2008-06-08 00:52:55Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3288 3288
type__enhancement enhancement
reporter__davep davep
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1212886358000000 1212886358000000
changetime__1213543437000000 1213543437000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set status to `closed`|
|-|-|

(In [23920]) Apply 0001-Fix-example.patch

Author: davep
Reviewer: exarkun
Fixes: [#3288](https://github.com/twisted/twisted/issues/3288)

Change the ampserver example to always do float division and to use the
correct name when returning the result of the Divide command.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
